src/EventPayload.ts

```diff
@@ -113,6 +113,8 @@ export interface EventPayload {
   pages?: {
     /** The scope of an action. Applies to Pages events. */
     scope?: string;
+    /** The legacy identifier of the event. */
```

Comment:
let's not use legacy as it indicates some connotation with the event.

Reply:
Ack